apps/editing-toolkit/editing-toolkit-plugin/full-site-editing-plugin.php

```diff
@@ -321,7 +321,7 @@ function load_wpcom_block_editor_sidebar() {
  */
 function load_coming_soon() {
 	if ( defined( 'WPCOM_PUBLIC_COMING_SOON' ) && WPCOM_PUBLIC_COMING_SOON ) {
-		require_once __DIR__ . '/coming-soon/coming-soon.php';
+		require_once __DIR__ . '/wpcom-coming-soon/class-wpcom-coming-soon.php';
```

Comment:
It looks like this PR is renaming the `coming-soon` dir to `wpcom-coming-soon`, which is fine, but I don't see anything in the PR that moves the existing files in the `coming-soon` folder. Does this change play nicely with the current coming soon v2 changes?

Reply:
How do people feel about renaming this dir to `./coming-soon-v2` and dropping the idea the wpcom prefix means something?

If not something like this probably works:

```
function load_coming_soon() {
	if ( defined( 'WPCOM_PUBLIC_COMING_SOON' ) && WPCOM_PUBLIC_COMING_SOON ) {
		require_once __DIR__ . '/coming-soon/coming-soon.php';
	} elseif ( defined( 'WPCOM_PUBLIC_COMING_SOON_V2' ) && WPCOM_PUBLIC_COMING_SOON_V2 ) {
		require_once __DIR__ . '/wpcom-coming-soon/class-wpcom-coming-soon.php';
	}
}
```

Reply:
The `WPCOM_PUBLIC_COMING_SOON` flag and the `coming-soon` directory are all part of the coming soon v2 effort. There's no coming soon v1 code in etk.

Reply:
In saying that I don't mind adding the `v2` suffix, but yeah we don't need to conditionally load one or the other. All the code should be in either `coming-soon` or `coming-soon-v2`.

Reply:
Before I forget to circle back - I rolled my changes into coming-soon from wpcom-coming-soon and got rid of the wpcom-coming-soon from the original PR.